docs/manual/grelfunctions.md

```diff
@@ -591,6 +591,9 @@ Returns a boolean indicating whether o has a member field called [name](expressi
 ###### coalesce(o1, o2, o3, ...) {#coalesceo1-o2-o3-}
 Returns the first non-null from a series of objects. For example, `coalesce(value, "")` would return an empty string “” if `value` was null, but otherwise return `value`.
 
+###### levenshteinDistance(s1, s2) {#levenshteinDistance-s1-s2}
+Returns how many single-character changes you need to make to turn one string into another. This includes adding characters, removing characters, and changing characters to match between the two strings. For example, `levenshteinDistance("New York", "newyork")` will return 1 and `levenshteinDistance("M. Makeba", "Miriam Makeba")` will return 5
```

Comment:
We should be clear what is returned. So e.g. "Returns an integer indicating how many single-character ..."